week2/Homework2/ListR.rb

```diff
@@ -0,0 +1,29 @@
+array = []
+alength = array.length
+
+leave = 0
+
+while leave = 0 do
+alength = array.length
+x = 0
+y = 1
+print "Items on list"
+
+while x <= alength do
+print y + "." + array[x]
+x = x + 1
+y = y + 1
+end
+
+puts "Add item to list?(Yes or No)"
+yn = gets
+  if yn = "yes" // yn= "Yes"
```

Comment:
To do you, you use `||`. Comparison is using two `=` signs. So you have to do this.

```rb
name = "Jaedon"
if name == "Jaedon"
  // do something
end
```